site/content/contribute/expectations/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Section 3 — Contributor expectations"
+heading: "Section 3 — Contributor expectations"
+description: ""
+date: "2022-10-01T12:00:00-00:00"
+weight: 3
+---
+
+## Subsection 1 — Before contributing
+
+1. If you’re working on code or documentation, sign the [Contributor License Agreement](https://mattermost.com/mattermost-contributor-agreement/) so you can be added to the Mattermost [Approved Contributor List](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true). If you’ve included your mailing address in the signed agreement, you might receive a [Limited Edition Mattermost Mug](https://forum.mattermost.com/t/limited-edition-mattermost-mugs/143) as a thank you gift after your first pull request is merged.
+2. For features that don’t belong as part of a core Mattermost repository:
+    1. You could [create an app]({{< ref "/integrate/apps/" >}}), which connects to Mattermost’s functionality while being hosted elsewhere.
+    2. External applications that don’t rely on customizing the Mattermost user experience can reach the data they need through [incoming]({{< ref "/integrate/webhooks/incoming" >}}) and [outgoing]({{< ref "/integrate/webhooks/outgoing" >}}) webhooks or [the API](https://api.mattermost.com/).
+    3. External functionality activated from within Mattermost might work best as a [slash command]({{< ref "/integrate/slash-commands/" >}}).
+    4. [Plugins]({{< ref "/integrate/plugins/" >}}) enable you to modify the fabric of the Mattermost UI/UX completely but come with the highest level of development overhead and must be written in Go and React.
+    5. If you need to take Mattermost with you into another app, perhaps you’re looking for [the embed guide]({{< ref "/integrate/customization/embedding/" >}}).
```

Comment:
Given that the preceding 4 steps are specific ways to contribute to areas of Mattermost beyond core repos, this line feels more like a tip or note rather than a fifth contribution opportunity.

Reply:
Please see my suggestion below; this may resolve it

Reply:
```suggestion
    5. If you need to take Mattermost with you into another app, perhaps you’re looking for [the embed guide]({{< ref "/integrate/customization/embedding/" >}}).
```